app/frontend/src/features/donations/constants.ts

```diff
@@ -0,0 +1,49 @@
+export const DONATIONS_BANNER_TITLE =
+  "Your help is needed to support our mission";
+export const DONATIONS_BANNER_TEXT =
+  "Your donation will support the future of couchsurfing";
+export const DONATIONS_TITLE = "You can build the future of couchsurfing";
```

Comment:
I think couch-surfing here needs a hyphen? Otherwise we might get into trouble with Couchsurfing.org 🤔

Reply:
> This looks really great! Thanks for doing this.
> 
> A few suggestions:
> 
>     1. Logo should be without the beta sticker
> 
>     2. Make the benefactor text its own variable
> 
>     3. Add a header donations_title2 "Where your donation will go" before the rest of donations_text2
> 
> 
> For mobile view, the order should be reorganised
> 
>     1. donation box
> 
>     2. benefactor text
> 
>     3. donations_title
> 
>     4. donations_text
> 
>     5. donations_title2
> 
>     6. donations_text2
> 
> 
> I'll also send through updated copy for the text sections

@ItsiW done the reordering in mobile. About the logo, I used our logo component as if I put an image and then the logo updates, we would also have to update it here manually, and that's not ideal. Maybe we could hide the beta sticker with a prop, what do you think, @lucaslcode ?